main.go

```diff
@@ -168,6 +173,23 @@ func isBitriseCacheAPIURL(url string) bool {
 	return url == os.Getenv("BITRISE_CACHE_API_URL")
 }
 
+func writeCachePullTimestamp() error {
+	f, err := os.Create(cachePullEndTimePath)
+
```

Comment:
NOTE: pls remove the extra line, error handling should happen right after the error appears